src/pages/integrations.astro

```diff
@@ -0,0 +1,215 @@
+      >
+        <option
+          value="all"
+          class="button-integration button-integration-active"
+        >
+          All
+        </option>
+        <option value="database" class="button-integration">
+          Database & <br />
+          Data Warehouse
+        </option>
+        <option value="language" class="button-integration">Language</option>
+        <option value="machine-learning" class="button-integration">
+          Machine Learning
+        </option>
+        <option value="messaging" class="button-integration">Messaging</option>
+        <option value="visualization" class="button-integration">
+          Visualization
+        </option>
+      </select>
+      <div class="grow grid grid-cols-2 sm:grid-cols-3 xl:grid-cols-4 gap-0.5">
+        {
+          integrations.map((integration) => (
+            <div
+              data-category={integration.category}
+              class="flex flex-col items-center justify-center outline outline-gray-100 h-36 md:h-64"
+            >
+              <a
+                class="flex flex-col items-center"
+                href={`/integrations/${integration.id}`}
+              >
+                <img
+                  class="h-[48px] md:h-[72px]"
+                  src={integration.logo}
+                  alt={`${integration.name} logo`}
+                />
+                <span class="text-gray-700 text-center font-medium text-base md:text-xl mt-2 md:mt-4">
+                  {integration.name}
+                </span>
+                <span class="text-gray-500 font-medium md:font-semibold text-center text-xs uppercase tracking-wider sm:tracking-widest mt-1">
+                  {integration.category}
+                </span>
+              </a>
+            </div>
+          ))
+        }
+      </div>
+      <div
+        class="lg:hidden mb-20 p-5 bg-blue-50 border rounded-lg border-blue-600"
+      >
+        <span class="font-semibold">Can't find an integration?</span>
+        <div class="mt-2 text-sm">
+          <span>
+            Integrations are hosted as open-source packages
+            <a
+              class="underline"
+              href="https://hex.pm/packages?search=kino&sort=recent_downloads"
+              target="_blank">on Hex.pm</a
+            >. Anyone can build and publish new integrations. Use our
+            <a
+              class="underline"
+              href="https://github.com/livebook-dev/livebook/discussions/categories/integrations"
+              target="_blank">GitHub Discussions</a
+            >
+            to suggest new integrations and discuss with the community.
+          </span>
+        </div>
+      </div>
+    </div>
+  </section>
+</Layout>
+
+<style>
+  .button-integration {
+    @apply hover:bg-gray-200 focus:bg-gray-200 active:bg-gray-300 text-left px-5 py-2 rounded-lg;
+  }
+
+  .button-integration-active {
+    @apply bg-gray-200;
+  }
+</style>
+
+<script>
```

Comment:
Note that the content of the `<script>` tag is properly compiled/transpiled into its own file. Same with `<style>`.